source/translator.d

```diff
@@ -0,0 +1,34 @@
+private string[string] defaultLanguage;
+static this()
+{
+	defaultLanguage = [
+		"run": "Run",
+		"reset": "Reset",
+		"keyboard_shortcuts": "Keyboard Shortcuts",
+	];
```

Comment:
For now I went with KISS and put the keys directly into the file. I doubt that we will need many, but we can always put the default keys in a separate file.